### ISSUE TITLE
Does anyone know how to use the value minResolution in textures.json ?

### ISSUE THREAD

**Issue by sunshuwei** [Stellarium/stellarium#4000]
The User Guide did not explain this clearly, and I did not find any effect after modifying the values.

User Guide: “minResolution UNDOCUMENTED VALUE!” 
Does it mean that this is an arbitrary value?

**Comment by gzotti** [comment]
No. It just means that the first team who have introduced this value (2007-09?) have not explained what they did in code comments for the MultiLevelJsonBase and StelSkyImageTile classes and and, we, the current team, have not reverse-engineered the exact internals of the StelSkyImageTile class to understand all the details that went into developing these important classes. If you can, please find out, and when you are sure, add documentation. I am sure there is a meaning in this. 

It is not completely undocumented, though. In StelSkyImageTile.hpp we find: 

`//! Minimum resolution of the data of the texture in degree/pixel
`

But I did not follow up to the end what depends on it.

**Comment by 10110111** [comment]
Looks like it's the resolution when the next-level tiles should be shown, see [this part of the code](https://github.com/Stellarium/stellarium/blob/71fb6dd587ed17e29a8ea010e117cb288ba78172/src/core/StelSkyImageTile.cpp#L230).

**Comment by gzotti** [comment]
Ah, right. Is this then relevant for DSS/HiPS only, or are our DSO textures also structured in this way (or are the overlapping DSO images in the DSO images issue thread a result of negligence of correctly configuring this value in the JSON?) I had the impression these classes were made to allow more than is now used. (e.g. what is dataSetCredits for?)

**Comment by gzotti** [comment]
And if this is property of the actual texture, why is it called minResolution, not just resolution?

**Comment by 10110111** [comment]
I don't know how the DSO textures are structed, but judging by the locations of the overlapping DSO images I suppose that they won't align well with HiPS tiles.

> And if this is property of the actual texture, why is it called minResolution, not just resolution?

It's the minimum "resolution" of the display after which tiles are switched. The "resolution" word here is a misnomer, since it's a reciprocal of the actual resolution: the larger the value, the poorer the image.

**Comment by sunshuwei** [comment]
Whether it represents the arc represented by each pixel in the image, also known as pixel scale.
